bugsnag/notification.py

```diff
@@ -37,17 +37,12 @@ def request():
             status = resp.getcode()
 
             if status != 200:
-                bugsnag.log("Notification to %s failed, status %d" % (url,
-                                                                      status))
+                bugsnag.logger.warning(
```

Comment:
This makes more sense. I considered changing this as a part of an unrelated changeset, though held off for a proper retuning of the logging behavior.